.dockerignore

```diff
@@ -0,0 +1,7 @@
+.git
+.github
+.gitignore
+.cache
+1.14.5/
+tests/
+tools/
```

Comment:
why?

Reply:
removed. disregard for now, i need to do more research.